src/routing/comm.rs

```diff
@@ -65,12 +64,16 @@ impl Comm {
         transport_config: qp2p::Config,
         event_tx: mpsc::Sender<ConnectionEvent>,
     ) -> Result<(Self, SocketAddr)> {
-        let quic_p2p = QuicP2p::with_config(Some(transport_config), &[], true)?;
+        let quic_p2p = QuicP2p::with_config(Some(transport_config), &[], true)
+            .map_err(|_| Error::InvalidConfig)?;
```

Comment:
but put an error log here to record the error from qp2p ?
same comment to the below